apps/src/metrics/AnalyticsConstants.js

```diff
@@ -116,7 +116,7 @@ const EVENTS = {
   AFE_HOMEPAGE_BANNER_SUBMIT: 'AFE Teacher Homepage Banner Submitted',
 
   // Sections
-  COMPLETED_EVENT: 'Section Setup Completed',
+  SECTION_SETUP_COMPLETED: 'Section Setup Completed',
```

Comment:
Much better name for the variable.

Reply:
I do like the you removed EVENT from the variable name as all these variables are under `EVENTS`.